packages/@aws-cdk/aws-scheduler-alpha/lib/schedule.ts

```diff
@@ -59,6 +59,47 @@ export interface ScheduleTargetProps {
   readonly retryAttempts?: number;
 }
 
+/**
+ * A time window during which EventBridge Scheduler invokes the schedule.
+ */
+export class FlexibleTimeWindowMode {
+  /**
+   * FlexibleTimeWindow is disabled.
+   */
+  public static off(): FlexibleTimeWindowMode {
+    return new FlexibleTimeWindowMode('OFF');
+  }
+
+  /**
+   * FlexibleTimeWindow is enabled.
+   */
+  public static flexible(maximumWindowInMinutes: Duration): FlexibleTimeWindowMode {
```

Comment:
```suggestion
  public static flexible(maxWindow: Duration): FlexibleTimeWindowMode {
```

It doesn't need to be in minutes anymore cuz we're supplying a Duration